src/pages/products/add/index.tsx

```diff
@@ -1,43 +1,58 @@
-import { useState } from "react";
+import { ChangeEvent, useState } from "react";
 import UploadProduct from "./style";
 import useInput, { ReturnType } from "../../../hooks/useInput";
-import { Category } from "../../../types/type";
+import { Category, UploadReturnObj } from "../../../types/type";
+import fileUpload from "../../../utils/upload";
 
 export interface UploadProductProps {
-  category: Category;
+  category: ReturnType;
   name: ReturnType;
   quantity: ReturnType;
-  description: ReturnType;
+  description: (value: string) => void;
   price: ReturnType;
   discount: ReturnType;
   shipping: ReturnType;
-  images: Array<Blob>;
+  images: UploadReturnObj[] | [];
+  handleImage: (ev: ChangeEvent<HTMLInputElement>) => void;
 }
 
 export default function UploadProductContainer() {
-  const [category, setCategory] = useState<Category>("beauty");
+  const category = useInput<Category>("beauty");
   const name = useInput("");
   const quantity = useInput(0);
-  const description = useInput("");
+  const [contents, setContents] = useState("");
 
   //price Info
   const price = useInput(0);
   const discount = useInput(0);
   const shipping = useInput(0);
 
   //image
-  const [images, setImages] = useState<Array<Blob>>([]);
+  const [images, setImages] = useState<Array<UploadReturnObj>>([]);
+
+  const handleImage = async (ev: ChangeEvent<HTMLInputElement>) => {
+    const target = ev.target;
+    const files = target.files! as FileList;
+    const image = await fileUpload(files);
+
+    setImages((images) => [...images, image]);
+  };
+
+  const handleContents = (value: string) => {
+    setContents(value);
+  };
 
   return (
     <UploadProduct
       category={category}
       name={name}
       quantity={quantity}
-      description={description}
+      description={handleContents}
```

Comment:
```suggestion
      handleContents={handleContents}
```

p5 : 아래의 handleImage처럼 어떤 prop인지 알 수 있게 이 부분도 이렇게 전달해주면 어떨까요?

Reply:
에디터 이슈로 이미 반영완료! 추가적으로 저도 파일에 productlist라는 걸 쓰고 있는데 지승님도 같은 파일이 있는지 그 부분에서 충돌나는데 확인이 필요합니다.